### PR TITLE
PCHR-3175: Add margin to report filters header

### PR DESCRIPTION
**Pull request by reneolivo**
## Overview
This PR adds a margin to the header of filters in SSP report pages.

## Before
leave reports hr17 9000 7

## After
leave reports hr17 9000 5

## Technical Details
Form groups were defined be displayed as inline elements with no margin bottom [here](https://github.com/compucorp/civihr-employee-portal-theme/blob/ddd1038efda89cfc102fbaa3508c92aa31820df0/civihr_default_theme/scss/base/_custom.scss#L1176):
```css
.form-group {
  display: inline-block;
  margin-bottom: 0;
  vertical-align: middle;
}
```

To be able to apply the margin for the filter's header, the rules stated above were overridden for `header.form-group` elements:

```css
header.form-group {
  display: block;
  margin-bottom: 1em;
}
```

Note: both definitions are only in the context of `#edit-absence-date-filter-wrapper` and `#edit-between-date-filter-wrapper` elements.

### scss/base/_custom.scss refactoring

Lint issues were fixed in the [scss/base/_custom.scss](https://github.com/compucorp/civihr-employee-portal-theme/blob/ce644a1eb65257b4fffc650cbdf2bc536cc23b98/civihr_default_theme/scss/base/_custom.scss) file.

Worth noting is the removal of these two rules since they were not adding anything:
[.table-responsive](https://github.com/compucorp/civihr-employee-portal-theme/pull/297/files#diff-408fa5bab1ade4518c7b975a457a71b3L1155):
pchr 3175 add margin to report filters header by reneolivo pull request 297 compucorp civihr employee portal theme

[label:](https://github.com/compucorp/civihr-employee-portal-theme/pull/297/files#diff-408fa5bab1ade4518c7b975a457a71b3L656)
pchr 3175 add margin to report filters header by reneolivo pull request 297 compucorp civihr employee portal theme 1

## Comments

This PR is related to https://github.com/compucorp/civihr-employee-portal/pull/419 since it gives a margin to the header labels introduced by that other PR

### CODE REVIEW

**Comment by igorpavlov-zz** [comment]
@reneolivo 
when you say "This was overridden for header.form-group elements" 
I am not sure if it was like that **before** or **after** your PR. Could you please amend this sentence to explicitly say if it was a part of the solution.

**Comment by igorpavlov-zz** [comment]
@reneolivo could you please clarify **how** this PR is related to another PR mentioned in the comment section.

File: civihr_default_theme/scss/base/_custom.scss [hunk 1]
```diff
@@ -1178,6 +1178,11 @@ form {
       margin-bottom: 0;
       vertical-align: middle;
     }
+
```
Comment (igorpavlov-zz) [inline civihr_default_theme/scss/base/_custom.scss:1268]: This file indentation, line spacing, comments styling (comments should be capitalised) are wrong. Could you please fix it in this PR?

**Review by igorpavlov-zz — CHANGES_REQUESTED** [review]

**Comment by igorpavlov-zz** [comment]
@reneolivo I also notice on the screenshot there is a 1px offset between selector and icon button. Is this expected?

**Comment by igorpavlov-zz** [comment]
@reneolivo 

I assume you made these things here:
1) removed empty lines (-1)
2) added empty lines (+1)
3) changed indentation (-1/+1)
4) ordered styling (-1/+1)

```
111 new empty lines
10  removed empty lines
551 total deletions
650 total additions

551 - 10 + 111 = 652
652 - 650 = 2 (?)
```

Do you have an idea what could these 2 lines be?

**Comment by reneolivo** [comment]
@igorpavlov I removed a CSS rule that had a commented `display: none` inside:

https://github.com/compucorp/civihr-employee-portal-theme/pull/297/commits/ce644a1eb65257b4fffc650cbdf2bc536cc23b98#diff-408fa5bab1ade4518c7b975a457a71b3L703

Maybe that's the reason.

**Review by igorpavlov-zz — APPROVED** [review]

**Pull request merged**